tools/ci_connector_ops/ci_connector_ops/qa_checks.py

```diff
@@ -93,9 +105,19 @@ def check_connector_https_url_only(connector: Connector) -> bool:
     Returns:
         bool: Wether the connector code contains only https url.
     """
-    # TODO implement
+    files_with_http_url = set()
+    for filename, line in read_all_files_in_directory(connector.code_directory, path_patterns_to_ignore):
+        if "http://json-schema.org/draft-07/schema#" in line:
+            continue
+        if "http://" in line:
+            files_with_http_url.add(str(filename))
```

Comment:
Yeah, I guess this all we need to check! 
Have you seen any false positives other than JSONschema?

Reply:
A malicious person could do `"h"+"t"+"t"+"P"+":"`, and that is a known risk that I'm comfortable with, given the firewalls we will be adding.

Reply:
> Have you seen any false positives other than JSONschema?

I've seen a couple in test folders, expected records etc. This is why I ignored any path having `test` in it.